### DISCUSSION TITLE
Welcome to AWS-Machine-Learning-Foundations-Scholarship Discussions!

### DISCUSSION THREAD

**Discussion by Shivansh2407** [Shivansh2407/AWS-Machine-Learning-Foundations-Scholarship#1] [Announcements]
We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we build together :muscle: .

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by irizwaririz** [comment]
Hi everyone, I think it would be useful to make some self-testing markdown files for each lesson.

We can follow the general idea of this: [Azure Machine Learning Program Self-Test Questions](https://www.notion.so/Azure-Machine-Learning-Program-Self-Test-Questions-962c2f25202d4f0c8903ac69fc5e5e5d)

And use this to add spoilers to our files (to hide the answers while self-testing): [GitHub spoiler markdown](https://gist.github.com/jbsulli/03df3cdce94ee97937ebda0ffef28287)

What do you guys think? 🤔

**Reply by Shivansh2407** [reply]
That's an awesome idea @xxprime ! Will work on it as well 🎉

**Comment by SSR-dev-hub** [comment]
Hi,

I am thinking of doing this program this year and wondering how it went last year with you all.
Here are my questions that could help others thinking to pursue this course.

1. Did you enjoy the program
2. Was it helpful for your career ?
3. Can this be any good for regular SWE to shift into the ML world
4. Any chance AWS will reach out to you after completing this nano degree for a job offer?

TIA

**Reply by irizwaririz** [reply]
Hello, sharing my insights below:

1. Yes, the concepts are presented very well and in an engaging manner
2. Yes. I was very lucky to be recruited into a side project, within my company, dealing with ML (specifically time series anomaly detection) and the basic concepts I learned in lesson 2 was very helpful in this case
3. This course will not be enough for you to shift into the ML world, you will need practical/industry experience for that. But it gives you the foundations to build upon so that you can learn the next layer of concepts and so that you can actually implement and practice with ML models
4. I'm not quite sure. But if I remember correctly, if you were one of the top graduates of the course, the course team will help give feedback on your resume. But in any case, nothing is impossible*
